### ISSUE TITLE
[Add/update publication] An Empirical Survey of the Effectiveness of Debiasing Techniques for Pre-trained Language Models

### ISSUE THREAD

**Issue by ncmeade** [McGill-NLP/mcgill-nlp.github.io#65]
### Action

Add publication

### Title

An Empirical Survey of the Effectiveness of Debiasing Techniques for Pre-trained Language Models

### Shorthand

2110.08527

### Author

Nicholas Meade

### Names

Nicholas Meade, Elinor Poole-Dayan, Siva Reddy

### Venue

ACL 2022

### Link

https://arxiv.org/abs/2110.08527

### Year

2022

### Month

05

### Day

22

### Tags

ACL

### Code

https://github.com/McGill-NLP/bias-bench

### Webpage

https://mcgill-nlp.github.io/bias-bench/

### Twitter

https://twitter.com/ncmeade/status/1511052940022267909?cxt=HHwWioCy7ZOpq_gpAAAA

### Thumbnail

https://user-images.githubusercontent.com/26423685/167192281-14d27c16-22e3-48a6-8d3a-78f6fac275ab.png

### Abstract

Recent work has shown pre-trained language models capture social biases from the large amounts of text they are trained on. This has attracted attention to developing techniques that mitigate such biases. In this work, we perform an empirical survey of five recently proposed bias mitigation techniques: Counterfactual Data Augmentation (CDA), Dropout, Iterative Nullspace Projection, Self-Debias, and SentenceDebias. We quantify the effectiveness of each technique using three intrinsic bias benchmarks while also measuring the impact of these techniques on a model's language modeling ability, as well as its performance on downstream NLU tasks. We experimentally find that: (1) Self-Debias is the strongest debiasing technique, obtaining improved scores on all bias benchmarks; (2) Current debiasing techniques perform less consistently when mitigating non-gender biases; And (3) improvements on bias benchmarks such as StereoSet and CrowS-Pairs by using debiasing strategies are often accompanied by a decrease in language modeling ability, making it difficult to determine whether the bias mitigation was effective.

**Comment by xhlulu** [comment]
For thumbnail can you use the direct link instead? Not the markdown formatted. You can edit the issue directly and remove & add the tag.

**Comment by ncmeade** [comment]
Sorry, fixed now
